sycl/source/detail/device_info.hpp

```diff
@@ -568,64 +568,122 @@ struct get_device_info_impl<range<Dimensions>,
+  X("5.0", oneapi_exp_arch::nvidia_gpu_sm_50)                                  \
+  X("5.2", oneapi_exp_arch::nvidia_gpu_sm_52)                                  \
+  X("5.3", oneapi_exp_arch::nvidia_gpu_sm_53)                                  \
+  X("6.0", oneapi_exp_arch::nvidia_gpu_sm_60)                                  \
+  X("6.1", oneapi_exp_arch::nvidia_gpu_sm_61)                                  \
+  X("6.2", oneapi_exp_arch::nvidia_gpu_sm_62)                                  \
+  X("7.0", oneapi_exp_arch::nvidia_gpu_sm_70)                                  \
+  X("7.2", oneapi_exp_arch::nvidia_gpu_sm_72)                                  \
+  X("7.5", oneapi_exp_arch::nvidia_gpu_sm_75)                                  \
+  X("8.0", oneapi_exp_arch::nvidia_gpu_sm_80)                                  \
+  X("8.6", oneapi_exp_arch::nvidia_gpu_sm_86)                                  \
+  X("8.7", oneapi_exp_arch::nvidia_gpu_sm_87)                                  \
+  X("8.9", oneapi_exp_arch::nvidia_gpu_sm_89)                                  \
+  X("9.0", oneapi_exp_arch::nvidia_gpu_sm_90)                                  \
+  X("gfx701", oneapi_exp_arch::amd_gpu_gfx701)                                 \
+  X("gfx702", oneapi_exp_arch::amd_gpu_gfx702)                                 \
+  X("gfx801", oneapi_exp_arch::amd_gpu_gfx801)                                 \
+  X("gfx802", oneapi_exp_arch::amd_gpu_gfx802)                                 \
+  X("gfx803", oneapi_exp_arch::amd_gpu_gfx803)                                 \
+  X("gfx805", oneapi_exp_arch::amd_gpu_gfx805)                                 \
+  X("gfx810", oneapi_exp_arch::amd_gpu_gfx810)                                 \
+  X("gfx900", oneapi_exp_arch::amd_gpu_gfx900)                                 \
+  X("gfx902", oneapi_exp_arch::amd_gpu_gfx902)                                 \
+  X("gfx904", oneapi_exp_arch::amd_gpu_gfx904)                                 \
+  X("gfx906", oneapi_exp_arch::amd_gpu_gfx906)                                 \
+  X("gfx908", oneapi_exp_arch::amd_gpu_gfx908)                                 \
+  X("gfx90a", oneapi_exp_arch::amd_gpu_gfx90a)                                 \
+  X("gfx1010", oneapi_exp_arch::amd_gpu_gfx1010)                               \
+  X("gfx1011", oneapi_exp_arch::amd_gpu_gfx1011)                               \
+  X("gfx1012", oneapi_exp_arch::amd_gpu_gfx1012)                               \
+  X("gfx1013", oneapi_exp_arch::amd_gpu_gfx1013)                               \
+  X("gfx1030", oneapi_exp_arch::amd_gpu_gfx1030)                               \
+  X("gfx1031", oneapi_exp_arch::amd_gpu_gfx1031)                               \
+  X("gfx1032", oneapi_exp_arch::amd_gpu_gfx1032)                               \
+  X("gfx1034", oneapi_exp_arch::amd_gpu_gfx1034)
+
+// This macro is only for Intel Device IPs
+#define INTEL_IPS(X)                                                           \
```

Comment:
Let's rename for consistency + comment above
```suggestion
#define INTEL_ARCHES(X)                                                           \
```